.scrutinizer.yml

```diff
@@ -0,0 +1,16 @@
+checks:
+    python:
+        code_rating: true
+        duplicate_code: true
+filter:
```

Comment:
Sorry! I have made a mistake about the repo, I was thinking about kytos-napps. This is ok!